docs/core-protocol/public-lock/transfers.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Transfers
+description: The Public Lock contract implements the ERC721 specification, including transfer functions, but their behavior can be customized!
+sidebar_position: 4
+---
+
+The Public Lock contract implements the [ERC721 specification](https://docs.openzeppelin.com/contracts/2.x/api/token/erc721), including transfer functions (`transferFrom`, `safeTransferFrom`, and the related `approve` and `setApprovalForAll`), but their behavior can be customized.
+
+:::info
+Expired keys **can not** be transferred.
+:::
+
+## Key Managers
+
+Each NFT (called a key in the Unlock Protocol) has an `owner`. Additionally, it may have a `key manager`. The `key manager`, if set, is the address that **has the transfer rights** over this specific key.
+
+The key manager can be set at the time of [minting](./minting-keys.md), either on purchases or on airdrops. Lock managers can also change the key manager for any key.
+
+You can read more about the roles in the [contract management section](./access-control.md).
+
+By setting a custom key manager on a key, the lock manager can disable transfers for this key (or at least make them only possible by the key manager).
+
+:::info
+When a transfer happens using `transferFrom` or `safeTransferFrom`, the lock manager is reset.
+:::
+
+## Fees on Transfers
+
+Many NFTs have the concept of **royalties**. Lock managers can apply royalties to their locks, especially using a [royalties registry like that one](https://royaltyregistry.xyz/lookup). It is important to note that these royalties **cannot be systematically enforced** and rely on the adoption by the platforms and exchanges on which NFTs are traded.
+
+However, inside Unlock Keys, the contract includes a unique feature to monetize secondary market transactions. The contract includes the ability to "burn" time upon transfers.
+
+Let's take an example. A user, Jane, owns an NFT membership valid for ten more days. The contract includes a 10% fee on each transfer. If Jane transfers her NFT to Claire, Claire will receive the key, but when she receives it, the expiration will have been reduced to 9 days.
+
+This mechanism is fully on-chain, and there is no way to bypass it.
+
+### Disabling Transfers
+
+The lock also has a mechanism to fully disable transfers for _any_ key, regardless of any other consideration, by setting a transfer fee of 100%.
```

Comment:
perhaps add a sentence (or a link) on how to set transfer fees (esp. how to make it nontransferrable by setting a 100% transfer fee)